docs/content/preview/contribute/core-database/build-from-src-ubuntu.md

```diff
@@ -51,48 +51,117 @@ AlmaLinux 8 is the recommended Linux development platform for YugabyteDB.
 
 {{< /note >}}
 
+The following instructions are for Ubuntu 20.04 or Ubuntu 22.04.
+
 ## Install necessary packages
 
 Update packages on your system, install development tools and additional packages:
```

Comment:
```suggestion
Update packages on your system, and install development tools and additional packages as follows:
```

Reply:
This can be done for all platforms in a followup.